docs/tools/vdb_table/data/aperturedb.json

```diff
@@ -9,51 +9,51 @@
     "slug": "aperturedb"
   },
   "oss": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "partial",
+    "source_url": "https://hub.docker.com/r/aperturedata/aperturedb-community",
+    "comment": "ApertureDB community edition is free and a great way to test all the database features, locally."
   },
   "license": {
-    "value": "",
-    "source_url": "",
-    "comment": ""
+    "value": "Proprietary",
+    "source_url": "https://github.com/aperture-data",
+    "comment": "ApertureDB code is proprietary but we have open source clients, dataset, and applications repositories."
   },
   "dev_languages": {
     "value": [
-      ""
+      "c++"
     ],
-    "source_url": "",
+    "source_url": "https://github.com/aperture-data",
     "comment": ""
   },
-  "vector_launch_year": 0,
+  "vector_launch_year": 2021,
   "metadata_filter": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "full",
+    "source_url": "https://docs.aperturedata.io/concepts/Schema",
+    "comment": "Complete graph database support alongside vector search"
   },
   "hybrid_search": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "full",
+    "source_url": "https://docs.aperturedata.io/python_sdk/object_wrappers/Descriptors",
+    "comment": "Blending graph filtering with vector search. Easy to introduce search algorithms. You can search across multiple embedding sets as well (collections)"
```

Comment:
Hybrid search does not reference filtering capabilities, that's the field above. For field descriptions, please check the schema file or mouse over table headers on vdbs.superlinked.com

Schema file:
https://github.com/superlinked/VectorHub/blob/main/docs/tools/vdb_table/vendor.schema.json

Specifically for Hybrid search:
Built-in blending of the 2 lists (keyword & vector search results) being done natively within the DB using something like Reciprocal Rank Fusion.

Reply:
Here is where this is a bit text specific in my opinion. A vector db can work for image embeddings and hybrid would then be vector search + some attributes of images. But I see your example for reranking + fusion and that's something we don't yet do. I removed this.